packages/appium/test/setup.js

```diff
@@ -0,0 +1,6 @@
+// This configures @babel/register to look for a babel config in parent dir(s) of
```

Comment:
helpful comment 👍

Reply:
yes, my verbosity increases with the size of the headache that the problem gave me